content/hi/docs/reference/glossary/customresourcedefinition.md

```diff
@@ -0,0 +1,19 @@
+---
+title: कस्टम संसाधन परिभाषा
```

Comment:
कस्टम रिसोर्स डिफिनिशन 
Use the same  word in entire docs as you used in more. I think it will be better to use this कस्टम रिसोर्स डिफिनिशन